weaviate/rbac/models.py

```diff
@@ -1,43 +1,50 @@
 from abc import abstractmethod
 from dataclasses import dataclass
 from enum import Enum
-from typing import List, Optional, Sequence, TypedDict, Union
+from typing import List, Optional, Sequence, TypedDict, Union, cast
 
 from pydantic import BaseModel
 
 from weaviate.cluster.types import Verbosity
 
 
-class PermissionBackup(TypedDict):
+class PermissionData(TypedDict):
+    collection: str
+    object_: str
```

Comment:
The value returned by the API is `object` so, despite the fact that it's shadowing here, this will need to be `object`. The original comment was not accurate!